### DISCUSSION TITLE
how to solve this problem, cp949

### DISCUSSION THREAD

**Discussion by dlatjdwns25** [3b1b/manim#1570] [Q&A] [answered]
`from os import *
from manim import *

class MathematicalEquationh(Scene):
    def construct(self):
        
        eq1=MathTex("2+4+6+...+2n")
        eq_s=MathTex("=")
        eq2=MathTex("n(n+1)")
        
        eq1.next_to(eq1, LEFT)
        eq2.next_to(eq2, RIGHT)

        eqgroup = VGroup(eq1,eq_s,eq2)

        self.play(write(eqgroup))
        
        self.wait(4)
`
--->

Manim Community v0.8.0

Traceback (most recent call last):
╭──────────────────────────────────────────────────────────────────────────────────────╮
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\c│
│li\render\commands.py", line 135, in render                                           │
│    132         for SceneClass in scene_classes_from_file(file):                      │
│    133             try:                                                              │
│    134                 scene = SceneClass()                                          │
│  ❱ 135                 scene.render()                                                │
│    136             except Exception:                                                 │
│    137                 error_console.print_exception()                               │
│    138                 sys.exit(1)                                                   │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\s│
│cene\scene.py", line 210, in render                                                   │
│     207         """                                                                  │
│     208         self.setup()                                                         │
│     209         try:                                                                 │
│  ❱  210             self.construct()                                                 │
│     211         except EndSceneEarlyException:                                       │
│     212             pass                                                             │
│     213         except RerunSceneException as e:                                     │
│ File "C:\manim\project\math11.py", line 7, in construct                              │
│     4 class Math(Scene):                                                             │
│     5     def construct(self):                                                       │
│     6                                                                                │
│  ❱  7         eq1=MathTex("2+4+6+⋅⋅⋅+2n")                                            │
│     8         eq_s=MathTex("=")                                                      │
│     9         eq2=MathTex("n(n+1)")                                                  │
│    10                                                                                │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\m│
│object\svg\tex_mobject.py", line 281, in __init__                                     │
│    278                         """                                                   │
│    279                     )                                                         │
│    280                 )                                                             │
│  ❱ 281             raise compilation_error                                           │
│    282         self.set_color_by_tex_to_color_map(self.tex_to_color_map)             │
│    283                                                                               │
│    284         if self.organize_left_to_right:                                       │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\m│
│object\svg\tex_mobject.py", line 259, in __init__                                     │
│    256         tex_strings = self.break_up_tex_strings(tex_strings)                  │
│    257         self.tex_strings = tex_strings                                        │
│    258         try:                                                                  │
│  ❱ 259             SingleStringMathTex.__init__(                                     │
│    260                 self,                                                         │
│    261                 self.arg_separator.join(tex_strings),                         │
│    262                 tex_environment=self.tex_environment,                         │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\m│
│object\svg\tex_mobject.py", line 85, in __init__                                      │
│     82                                                                               │
│     83         assert isinstance(tex_string, str)                                    │
│     84         self.tex_string = tex_string                                          │
│  ❱  85         file_name = tex_to_svg_file(                                          │
│     86             self.get_modified_expression(tex_string),                         │
│     87             environment=self.tex_environment,                                 │
│     88             tex_template=self.tex_template,                                   │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\u│
│tils\tex_file_writing.py", line 46, in tex_to_svg_file                                │
│     43     if tex_template is None:                                                  │
│     44         tex_template = config["tex_template"]                                 │
│     45     tex_file = generate_tex_file(expression, environment, tex_template)       │
│  ❱  46     dvi_file = compile_tex(                                                   │
│     47         tex_file, tex_template.tex_compiler, tex_template.output_format       │
│     48     )                                                                         │
│     49     return convert_to_svg(dvi_file, tex_template.output_format)               │
│ File "c:\users\a0103\appdata\local\programs\python\python39\lib\site-packages\manim\u│
│tils\tex_file_writing.py", line 191, in compile_tex                                   │
│    188                     "Check your LaTeX installation."                          │
│    189                 )                                                             │
│    190             with open(log_file, "r") as f:                                    │
│  ❱ 191                 log = f.readlines()                                           │
│    192                 error_pos = [                                                 │
│    193                     index for index, line in enumerate(log) if line.startswith│
│    194                 ]                                                             │
╰──────────────────────────────────────────────────────────────────────────────────────╯
UnicodeDecodeError: 'cp949' codec can't decode byte 0xe2 in position 1001: illegal multibyte sequence`

how..?

**Comment by TonyCrane** [comment, marked as answer]
It seems that you are using the community version of manim.
So you are supposed to ask questions in the discord channel, or open an issue in [ManimCommunity/manim](https://github.com/ManimCommunity/manim) instead of here

**Reply by dlatjdwns25** [reply]
What is the difference between the two?

**Reply by TonyCrane** [reply]
There are a lot of differences. See there document: [docs.manim.community](https://docs.manim.community)
